sql/telemetry_derived/clients_histogram_aggregates_merged_v1/query.sql

```diff
@@ -1,14 +1,24 @@
+WITH old_data AS
+  (SELECT *
+  FROM telemetry_derived.clients_hist_aggs_old_with_sample
+  WHERE sample_id >= @min_sample_id
+    AND sample_id <= @max_sample_id),
+
+new_data AS
+  (SELECT *
+  FROM telemetry_derived.clients_hist_aggs_new_with_sample
+  WHERE sample_id >= @min_sample_id
+    AND sample_id <= @max_sample_id)
+
```

Comment:
I moved these out of the `where` clause below because the partitions didn't seem to get used effectively (BQ listed more data being processes) when they were in the `where` clause